proposals/NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md

```diff
@@ -0,0 +1,75 @@
+# Handling VR help requests when application does not send VR help prompt
+
+* Proposal: [SDL-NNNN](NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md)
+* Author: [Irina Getmanets](https://github.com/GetmanetsIrina)
+* Status: **Awaiting review**
+* Impacted Platforms: Core, RPC
+
+## Introduction
+
+"vrHelp" items and "helpPrompt" are set by SetGlobalProperties for helping user to make voice navigation over registered application, what is very useful in case your hands are busy with driving.
+Safety requirements are very strong in automotive, so possibility to use Voice Recognition should be provided for driver all the time.
+
+## Motivation
+
+If there is no SetGlobalProperties issued, the default help item list will be populated by command list shared by Voice module or by default values from .ini file. If application issues a SetGlobalProperties with help item list, then this list will have precedence and it will be displayed on HMI.
```

Comment:
@GetmanetsIrina 
- I think it should be "_the default help items list will be populated with command list sent to voice module or with default values from smartDeviceLink.ini (link to the line in the file) file_"
- _then this list will have precedence over default values ..._
- _The purpose of proposal is to implement logic for default values of ..._